src/utils/hiddenRequire.js

```diff
@@ -0,0 +1,12 @@
+/*
+  Since webpack relies on search and replace of require 'string' or require "string".
+  Calling require via another function hides the requirement from webpack/browserify.
+  I found this out originally when I tried making a requires iterator which did not work.
```

Comment:
Awesome to know the backstory. Can we remove the last sentence since it is a personal anecdote more than docs? It was nice to see though, those types of mistakes are how I learn as well 😄.